### DISCUSSION TITLE
Make it runnable in all JS environments (Cloudflare Workers, ...)

### DISCUSSION THREAD

**Discussion by mnlbox** [diegomura/react-pdf#2713] [Ideas]
Hi @diegomura,
First thanks for this awesome library.
I have a question/suggestion: I just tried to use this library on the Cloudflare Workers and I resized that it's not runnable there as there is a dependency to node `fs` module. Is there any plan to make this library runnable on the edge environments as well?
It can be a nice feature.

**Comment by jacobgad** [comment]
+1 to this feature

**Comment by matbrgz** [comment]
Any idea how to start this implementation?
